OSRMTextInstructions/TokenType.swift

```diff
@@ -21,15 +22,17 @@ public enum TokenType: Int, CustomStringConvertible {
             type = .destination
         case "rotary_name":
             type = .rotaryName
+        case "exit":
+            type = .exitCode
         case "exit_number":
-            type = .exit
+            type = .exitIndex
```

Comment:
Per discussion with @bsudekum, we’re going to merge this and get a release out in order to unblock a navigation SDK release. However, if you think we should change the enumeration values back to match the JavaScript version, please open a ticket and we can address it in a subsequent release. (We’re pre-1.0, so we don’t have to worry much about backwards compatibility.)